easybuild/easyconfigs/n/NAMD/NAMD-2.13-foss-2019b-mpi.eb

```diff
@@ -0,0 +1,27 @@
+name = 'NAMD'
+version = '2.13'
+versionsuffix = '-mpi'
+
+homepage = 'http://www.ks.uiuc.edu/Research/namd/'
```

Comment:
@ekieffer the tests are failing because they now require `https` even in `homepage`